newsroom/templates/coverage_request_content.txt

```diff
@@ -0,0 +1,4 @@
+{% block content %}
+A coverage inquiry has been submitted by {{ name }} ({{ email }}) from {{ company }} regarding Agenda item: <a href="{{url}}">{{ url }}
```

Comment:
The link in this template should be plain text

Reply:
Why should a URL link be plain text?

Reply:
Yup, just realized it is a .txt. Will correct it. Thanks.